packages/ui/core/select/List.types.ts

```diff
@@ -0,0 +1,13 @@
+import { PropsWithClassName } from '../../types/core';
+
+export type SelectItem = {
+  name: string;
+  id: number;
+};
+
+export type SelectProp = PropsWithClassName & {
+  items: SelectItem[];
+  selected: number;
+  onChange: Function;
```

Comment:
You can now replace this for something like:
```suggestion
  onChange: (_id: string) => void;
```
The param type will depend on how you implemented the component